book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
Great job @AyomipoMajek 🤝

- [ ] Kindly create the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental`. Forexample a `book has many rentals`.
- [ ] Kindly ensure a method for adding rentals should be present. Follow this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more guidance.
- [ ] Also ensure the rentals variable is initialsed. You may also follow this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) for ore guidance.

Reply:
Done, thanks